spec/services/update_interviews_provider_spec.rb

```diff
@@ -44,6 +44,12 @@
           described_class.new(new_provider_params).save!
         }.to change(application_choice, :updated_at)
       end
+
+      it 'sends an email' do
```

Comment:
instead of enabling sidekiq and reaching into deliveries could we instead use the following strategy:
```
mailer = instance_double(ActionMailer::MessageDelivery, deliver_later: true)
allow(CandidateMailer).to receive(: interview_updated).and_return(mailer)
```

and in the expectation:
```
expect(CandidateMailer).to have_received(:interview_updated).with(application_choice)
```
what do you think?